packages/color-kit-block/src/Color.tsx

```diff
@@ -32,26 +32,32 @@ export const Color = ({ isEditing, color }: ColorProps): ReactElement => {
                 style={{
                     backgroundColor: toRgbaString(colorWithDecimalAlpha),
                 }}
-                className="tw-w-6 tw-h-6 tw-overflow-hidden tw-shadow-inner-line-y tw-transition-shadow hover:!tw-shadow-inner-line-strong"
+                className={merge([
+                    'tw-w-6 tw-h-6 tw-overflow-hidden tw-shadow-inner-line-y tw-transition-shadow',
+                    !isEditing && 'hover:!tw-shadow-inner-line-strong',
+                ])}
                 onClick={() => copy(`#${color.hex}`)}
             />
         </div>
     );
 
     return isEditing ? (
-        <ColorBox />
+        <div className="[&:first-child>div>div]:tw-shadow-inner-line-first [&:last-child>div>div]:tw-shadow-inner-line-last">
```

Comment:
extract to const className="[&:first-child>div>div]:tw-shadow-inner-line-first [&:last-child>div>div]:tw-shadow-inner-line-last"
 and pass that className
```suggestion
        <div className={className}>
```